manifests/02-rbac.yaml

```diff
@@ -90,6 +112,7 @@ rules:
   - create
   - get
   - update
+  - delete
```

Comment:
There were two blocks specifying access to clusteroperators, the first one used * for verbs (see line 24).